testing/ef-tests/src/macros/sanity_blocks.rs

```diff
@@ -1,95 +1,97 @@
 #[macro_export]
 macro_rules! test_sanity_blocks {
-    () => {
-        #[cfg(test)]
-        #[allow(non_snake_case)]
-        mod test_sanity_blocks {
-            use std::{fs, path::Path};
+    ($operation_name:ident, $path:expr) => {
+        paste::paste! {
+            #[cfg(test)]
+            #[allow(non_snake_case)]
+            mod $operation_name {
```

Comment:
```suggestion
    ($path:expr) => {
        paste::paste! {
            #[cfg(test)]
            #[allow(non_snake_case)]
            mod $path {
```

couldn't we just use the `path` as the mod name?

Reply:
Oh maybe the / is an issue? if that is the case then this is fine

Reply:
> Oh maybe the / is an issue? if that is the case then this is fine

I was running into an issue with the / so I did it this way